assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -194,6 +265,35 @@ export const reducer = ( state, { type, payload } ) => {
 			};
 		}
 
+		case FINISH_FETCH_CREATE_ACCOUNT:
+			return {
+				...state,
+				isFetchingCreateAccount: false,
+			};
+
+		case CATCH_FETCH_CREATE_ACCOUNT:
+			const { error } = payload;
+			return {
+				...state,
+				error,
+				isFetchingCreateAccount: false,
+			};
+
+		case START_FETCH_CREATE_ACCOUNT: {
+			return {
+				...state,
+				isFetchingCreateAccount: true,
+			};
+		}
+
+		case RECEIVE_CREATE_ACCOUNT: {
+			const { accountTicket: { id } } = payload;
+			return {
+				...state,
+				accountTicketTermsOfServiceURL: `https://analytics.google.com/analytics/web/?provisioningSignup=false#management/TermsOfService/&api.accountTicketId=${ id }`,
```

Comment:
Is this URL going to have the same problem as other deep links for users who sign in with multiple accounts?

That is, if you go to this URL but the user you've authenticated Site Kit with is not your primary logged in Google account (e.g. `authuser=1`) will it still work?

Reply:
Great question, we'll have to test that. Have we been able to fix that elsewhere at all?

Reply:
Not that I am aware of. As far as I know, we don't have a way to detect which index to use for `authuser`. I would expect that to maybe be available after going through the oAuth flow though 🤔

Reply:
I have found random threads describing that `authuser` (at least on certain services like Google Drive) also supports passing the Google account's email address. We should try if that works here.

While AdSense doesn't support the `authuser` query parameter and instead uses a `/u/{index}` affix in its URLs, I've already verified that at least there passing the email is supported - it will then internally redirect to the correct index number. So let's try it with the `authuser` parameter and email for Analytics.

Reply:
I'll give that a try, thanks!

Reply:
@felixarntz Brilliant! I verified this works as desired. I first tried the provisioning url with one google account while logged into another in Analytics and was shown an "Error: Access denied" screen. I then edited the url adding &authuser={my other email} - with this URL, Analytics redirected me to a login (as the authuser). when I completed that I reached the TOS screen correctly.

Do you think we should send the email address stored in the user profile along with the request from the plugin, or look the user up on the service/proxy side (and I assume use their first available email address)?

Reply:
Great find! I think we should pass the email through from the `Profile` as this is set after the user authenticates, so it should always match. This is also what we use for the login hint so I think it would make sense to use here since that's essentially what it is.

@adamsilverstein does the address need to be URL encoded or its fine as-is?

Reply:
@adamsilverstein @aaemnnosttv Since we now need the user's email address, this is theoretically blocked by #1176, which brings that data into a datastore. But we shouldn't be waiting for this here, as it would delay the work too much. Because whatever we'll do here will be a temporary solution, I think it's okay to just implement the minimal PHP-side part from #1176 (implement a `_googlesitekitUserData` global on the PHP side that outputs this information) - then we can directly access that global from here, and once #1176 is really implemented switch to using the selector.

Reply:
I confirmed that this works with Search Console as well! 😄 

I just opened a new issue to address this in our existing deep links here: https://github.com/google/site-kit-wp/issues/1456

Reply:
@adamsilverstein @aaemnnosttv Actually, let's do that in a follow-up PR though. Adding the email address to the URL shouldn't block this one from being completed - it wasn't a requirement in the first place and is rather an enhancement which we'll later need to apply to many areas.

Reply:
👍

Reply:
> does the address need to be URL encoded or its fine as-is?

In my test I used as is without encoding.